src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -979,6 +979,7 @@ public static bool IsDefaultExpressionContext(this SyntaxTree syntaxTree, int po
             int position,
             SyntaxToken tokenOnLeftOfPosition,
             CancellationToken cancellationToken,
+            ISet<SyntaxKind> allowedModifiersBefore = null,
```

Comment:
This seems like overkill.  I don't care if we happen to offer other parameter modifiers once we've offered ref/in/etc.